schema/transportation/segment.yaml

```diff
@@ -455,23 +526,23 @@ properties:
           properties: { is_at_most: { allOf: [ { "$ref": "#/$defs/propertyDefinitions/weightValueWithUnit" } ] } }
         - required: [is_less_than]
           properties: { is_less_than: { allOf: [ { "$ref": "#/$defs/propertyDefinitions/weightValueWithUnit" } ] } }
-    sequenceEntry:
+    subSegmentReference:
```

Comment:
One idea for name is `connectorOnSegment`.

Reply:
reverted, no longer needed after refactored the property to live on the 'from'